### ISSUE TITLE
Run on V100 16GB GPU?

### ISSUE THREAD

**Issue by htcml** [karpathy/nanoGPT#228]
Is it possible to pre-train the model using a V100 16GB GPU?

**Comment by toninog** [comment]
yes.  I used one and it worked.  I even tried with A4000 (laptop edition) as well as M2 Macbook
